src/components/icons/dark.icon.astro

```diff
@@ -0,0 +1,10 @@
+---
+export interface Props {
```

Comment:
I think the Props type must not be exported, if I remember the Astro docs correctly.

Reply:
yes, you remeber correctly 😁